src/monsters.cpp

```diff
@@ -220,9 +196,9 @@ std::list<Item*> MonsterType::createLootItem(const LootBlock& lootBlock)
 			tmpItem->setText(lootBlock.text);
 		}
 
-		itemList.push_back(tmpItem);
+		itemList.emplace_back(tmpItem);
```

Comment:
push_back was fine here, I'm not sure exactly what emplace_back does under the hood when using pointers (for objects it does in-place construction), I guess it differs depending on implementations, but in the best case it will do exactly what push_back does, in a worse case it will have performance overhead.